src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -805,6 +805,42 @@ HTTPS can be enabled for Web applications using a Java keystore. See `web.https`
 for the possible settings. We provided a default self-signed keystore than can be used for testing but for
 production you should build your own keystore with a valid certificate so it will be accepted by browsers.
 
+[[_script_engines]]
+== Configure script engines
+
+Most script engines do not need any configuration. This section talks about the script engines which can be configured
+individually.
+
+=== Docker-compose script engine
+
+The docker-compose script engine, is a wrapper around the docker-compose command. It needs the docker-compose
+executable and docker installed to work. It has a configuration file and can be configured individually for each ProActive Node.
+The configuration file is in config/scriptengines/docker-compose.properties.
+docker-compose.properties has following configuration options:
+
+---
+docker.compose.command=[docker-compose executable example:/usr/local/bin/docker-compose]
+---
+Defines which docker-compose executable is used by the script engine.
+
+---
+docker.compose.sudo.command=[sudo executable example:/usr/bin/sudo]
+---
+Defines the sudo executable which is used if configured so. The sudo property is used to give the docker-compose command root rights.
+
+---
+docker.compose.use.sudo=false
+---
+Defines whether to execute docker-compose with sudo [true] or without sudo [false]
+
+---
+docker.host=
+---
+Defines the DOCKER-HOST environment variable when executing docker-compose. The DOCKER_HOST variable defines how to connect to the
```

Comment:
Property docker host is named differently 3 times: DOCKER-HOST, DOCKER_HOST, DOCKER_host. Is it normal?